saltstack/base/salt/ccmv2-inverting-proxy-agent/cdp/bin/ccmv2/run-inverting-proxy-agent.sh

```diff
@@ -6,7 +6,6 @@
 SCRIPT=`basename "$0"`
 BACKEND_ID=$1
 LOG_PATH="/var/log/ccmv2-inverting-proxy-agent.log"
+CONFIG_FILE="/cdp/bin/ccmv2/config.toml"
 
-. /cdp/bin/ccmv2/inverting-proxy-agent-values-${BACKEND_ID}.sh
-
-exec /cdp/bin/ccmv2/inverting-proxy-agent -scheme ${SCHEME} -proxy ${INVERTING_PROXY_URL} -host ${BACKEND_ADDRESS} -backend ${BACKEND_ID} -disable-gce-vm-header=true -trusted-proxy-cert-path ${TRUSTED_PROXY_CERT_PATH} -trusted-backend-cert-path ${TRUSTED_BACKEND_CERT_PATH} -cert-path ${AGENT_CERT_PATH} -key-path ${AGENT_KEY_PATH} -http-proxy-url=${HTTP_PROXY_URL} > "$LOG_PATH" 2>&1
+exec /cdp/bin/ccmv2/inverting-proxy-agent -config-file ${CONFIG_FILE} > "$LOG_PATH" 2>&1
```

Comment:
I think this should be like `>> "$LOG_PATH"` to ensure the log is not overwritten

Reply:
Ok. I followed what was there already but the comment makes sense to me. Appending would be a better way of doing it since we do not want to lose the logs if the agent got restarted for some reason. I will make this change